lib/workers/bin/run_single_worker.rb

```diff
@@ -59,6 +59,12 @@
 require File.expand_path("../../../config/environment", __dir__)
 
 worker_class = worker_class.constantize
+missing_roles = ServerRole.where(:name => worker_class.required_roles) - MiqServer.my_server.active_roles
+unless missing_roles.empty?
+  puts "ERR:  Server roles are not sufficient for `#{worker_class}` worker. Missing: #{missing_roles.collect(&:name)}"
```

Comment:
Looks good then... maybe STDERR.puts or `warn`... some people are very picky about what goes to standard output  @NickLaMuro @kbrock

Reply:
I would go with `STDERR.puts` just because it is an "Error", not a warning (both might go to the same IO, haven't looked, but reads like it is going to be printed as an error that way).

Also, @jrafanie ... are you trying to suggest I am picky?!?